sdk/appconfiguration/azure-appconfiguration-provider/azure/appconfiguration/provider/_azureappconfigurationprovider.py

```diff
@@ -76,51 +78,45 @@ def load(
     refresh_interval: int = 30,
     on_refresh_success: Optional[Callable] = None,
     on_refresh_error: Optional[Callable[[Exception], None]] = None,
-    **kwargs
+    feature_flag_enabled: bool = False,
+    feature_flag_selectors: Optional[List[SettingSelector]] = None,
+    feature_flag_refresh_enabled: bool = False,
+    **kwargs,
 ) -> "AzureAppConfigurationProvider":
     """
     Loads configuration settings from Azure App Configuration into a Python application.
 
     :param str endpoint: Endpoint for App Configuration resource.
-    :param credential: Credential for App Configuration resource.
-    :type credential: ~azure.core.credentials.TokenCredential
-    :keyword selects: List of setting selectors to filter configuration settings
-    :paramtype selects: Optional[List[~azure.appconfiguration.provider.SettingSelector]]
-    :keyword trim_prefixes: List of prefixes to trim from configuration keys
-    :paramtype trim_prefixes: Optional[List[str]]
-    :keyword keyvault_credential: A credential for authenticating with the key vault. This is optional if
-     keyvault_client_configs is provided.
-    :paramtype keyvault_credential: ~azure.core.credentials.TokenCredential
-    :keyword keyvault_client_configs: A Mapping of SecretClient endpoints to client configurations from
-     azure-keyvault-secrets. This is optional if keyvault_credential is provided. If a credential isn't provided a
-     credential will need to be in each set for each.
-    :paramtype keyvault_client_configs: Mapping[str, Mapping]
-    :keyword secret_resolver: A function that takes a URI and returns a value.
-    :paramtype secret_resolver: Callable[[str], str]
-    :keyword refresh_on: One or more settings whose modification will trigger a full refresh after a fixed interval.
-    This should be a list of Key-Label pairs for specific settings (filters and wildcards are not supported).
-    :paramtype refresh_on: List[Tuple[str, str]]
+    :param  ~azure.core.credentials.TokenCredential credential: Credential for App Configuration resource.
+    :keyword Optional[List[~azure.appconfiguration.provider.SettingSelector]] selects: List of setting selectors to
+    filter configuration settings
+    :keyword Optional[List[str]] trim_prefixes: List of prefixes to trim from configuration keys
+    :keyword  ~azure.core.credentials.TokenCredential keyvault_credential: A credential for authenticating with the key
+    vault. This is optional if keyvault_client_configs is provided.
+    :keyword Mapping[str, Mapping] keyvault_client_configs: A Mapping of SecretClient endpoints to client
+    configurations from azure-keyvault-secrets. This is optional if keyvault_credential is provided. If a credential
```

Comment:
This is following the docstring style. If you look at the hover tooltip this isn't how it will look. Plus, I don't want to change it here, part of the auto-failover pr will be updating these to follow next-pylint which the SDK team is moving too.